src/main/java/edu/bu/metcs673/trackr/security/JwtFilter.java

```diff
@@ -157,6 +157,7 @@ private String getJwt(HttpServletRequest request) {
 		// If no header then check the cookie
 		if (StringUtils.isBlank(jwt)) {
 			Cookie cookie = WebUtils.getCookie(request, JWT_COOKIE_NAME);
+			cookie.setSecure(true);
```

Comment:
We need to remove as we are not creating the cookie here but just fetching it from the request in the filter.

Reply:
will do